aea/decision_maker/messages/transaction.py

```diff
@@ -50,7 +51,7 @@ class Performative(Enum):
     def __init__(
         self,
         performative: Performative,
-        skill_callback_ids: List[str],
+        skill_callback_ids: Sequence[Union[PublicId, str]],
```

Comment:
`Sequence` if far better than `List` for annotate types in methods. The same holds for other data structures.
[https://mypy.readthedocs.io/en/stable/common_issues.html#invariance-vs-covariance](https://mypy.readthedocs.io/en/stable/common_issues.html#invariance-vs-covariance)

Reply:
@marcofavorito this is something we should chat about with @5A11 on slack!